app/src/main/java/org/openobservatory/ooniprobe/activity/RunningActivity.java

```diff
@@ -46,7 +44,7 @@
 import localhost.toolkit.app.fragment.ConfirmDialogFragment;
 import localhost.toolkit.app.fragment.MessageDialogFragment;
 
-public class RunningActivity extends AbstractActivity implements ConfirmDialogFragment.OnConfirmedListener, ServiceConnection {
+public class RunningActivity extends AbstractActivity implements ConfirmDialogFragment.OnConfirmedListener {
```

Comment:
I'd invest a little more effort and add a one-line of documentation explaining the purpose of this class while we're touching it, such that our future selves more clearly know what this class is about.